examples/tests/test_01_provider_fastapi.py

```diff
@@ -30,11 +30,12 @@
 
 import pytest
 import uvicorn
-from examples.src.fastapi import app
 from pact import Verifier
-from pydantic import BaseModel
```

Comment:
Are these different projects, is this a typo?

- https://pypi.org/project/pydantic/
- https://pypi.org/project/pedantic/

Reply:
I... have no idea why that changed. Good cath, it should definitely still be `pydantic`. The only thing I can think of that caused this is an overly eager spell-checker 😓